tests/Kernel/Paragraphs/SocialMediaFollowTest.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Kernel\Paragraphs;
+
+use Drupal\paragraphs\Entity\Paragraph;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests the "Social media follow" paragraph.
+ */
+class SocialMediaFollowTest extends ParagraphsTestBase {
+
+  /**
+   * Tests the rendering of the paragraph type.
+   *
+   * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
+   */
+  public function testRendering(): void {
+    // Create social media link items.
+    $items = [
```

Comment:
To ease the readability, move this to a `getSocialMediaLinks` protected function . Also add two Twitter handles.

Reply:
Add also a test scenario where we have 1 type missing to check that it's not rendered.